### ISSUE TITLE
WEBP image could not be saved in 

### ISSUE THREAD

**Issue by MichaelOC05** [django-master/django-webp#43]
I am using Pillow 10.0.0, whitnoise 6.5.0, Django 4.2.3, django-webp 2.1.1.

When running locally with the no WEBP_CACHE the images are created in WEB_CACHE/images but are not accessible. Log output is Not Found: /static/WEBP_CACHE/filename. If I restart the application images that have been created in the WEB_CACHE/images directory are loaded, however images that have not been created in this directory will not load. They will load if the application is restarted again.

I was mostly able to fix this by creating a python command that looped through the images in the static directory and called the webp function in dajngo_webp.tempatetags.webp.py. 

When deployed if I run this custom command or if I attempt to use the package as it is I get and error WEBP image could not be saved in app/project/staticfiles/WEBP_CACHE/images/imagename. The deployment is to openshift.

**Comment by bimmui** [comment]
Hey Michael, 

I was able to replicate the error locally and I'm currently working on a solution to it. Could you provide a bit of info on how you have your static files set up in your production sever?

**Comment by MichaelOC05** [comment]
Yeah no problem, STATIC_URL = "static/"  STATIC_ROOT = BASE_DIR / "staticfiles" STATICFILES_DIR = [BASE_DIR / "static"]. Thanks for looking into this

**Comment by MichaelOC05** [comment]
Hi sorry I was wondering if you were able to make any headway with this issue?
